spec/railsapps/shared/all/app/mailers/auto_mailer.rb

```diff
@@ -1,4 +1,6 @@
 class AutoMailer < ActionMailer::Base
+  self.asset_host = 'http://asset.host.com'
```

Comment:
I think it's okay to have it being different types in the different mailers. As long as both paths have been crossed and we have some unit tests I think we're good.